components/ability-item.tsx

```diff
@@ -18,6 +18,11 @@ interface AbilityItemP {
   description: Ability["description"];
   feats: Ability["feats"] | PartialFeat[];
   usage: Ability["usage"];
+  version: Ability["version"];
+  variant: Ability["variant"];
```

Comment:
These may end up collapsed into each other.

The original design was that "version" is the rework of a class, while "variant" is a selectable option that heavily modifies its features (in particular, sorcerer bloodlines and specialist wizard in DATP). 

One benefit of keeping them separate would be displaying the variant selector somewhere more appropriate, e.g. next to the bloodline section for sorcerer.